mne/viz/evoked.py

```diff
@@ -642,6 +643,14 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
         data. Defaults to False.
     show : bool
         Show figure if True.
+    legend : bool
+        If True, automatically create a legend based on evoked.comment.
+    legend_loc : int | string | tuple
+        The location parameter given to the matplotlib legend call. For
+        example, 'upper right'. By default, the location is chosen
+        automatically.
+    legend_fontsize : int
+        The font size for the legend.
```

Comment:
Parameter bloat is bad indeed
Somehow I thought that @agramfort -1'd combining the `legend` and `legend_loc` parameters, but I cannot find that comment anymore, so maybe it happened in a dream.
So how does the parameter order go, you put new parameters at the end, but `show` is always last?